backend/box/box.go

```diff
@@ -399,7 +403,11 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 		return err
 	})
 
-	// Get rootID
+	// Get rootFolderID
```

Comment:
Can you add it to the `Options` struct like this

```go
type Options struct {
    // ...
    RootFolderID              string               `config:"root_folder_id"`
}
```

Instead of using `Get`, then you can use use `f.opt.RootFolderID` in place of `rootID`